aten/src/ATen/native/Gesv.h

```diff
@@ -2,17 +2,31 @@
 
 namespace at { namespace native {
 
-static inline void checkInputs(const Tensor& self, const Tensor& A) {
-  if (A.size(-1) != A.size(-2)) {
-    AT_ERROR("A must be batches of square matrices, "
-        "but they are %lld by %lld matrices",
-        (long long)A.size(-1), (long long)A.size(-2));
-  }
-  if (A.size(-1) != self.size(-2)) {
-    AT_ERROR("Incompatible matrix sizes for matmul: each A "
-        "matrix is %llu by %lld but each b matrix is %lld by %lld.",
-        (long long)A.size(-1), (long long)A.size(-1),
-        (long long)self.size(-2), (long long)self.size(-1));
+static inline void checkInputs(const Tensor& self, const Tensor& A, bool batched) {
+  if (batched) {
+    if (A.size(-1) != A.size(-2)) {
+      AT_ERROR("A must be batches of square matrices, "
+          "but they are %lld by %lld matrices",
```

Comment:
I don't think the formatting works. I faced an issue with this earlier.

Reply:
I wonder why they used this format specifier, lol, I remember seeing it didn't work but I thought it was because we use `setup.py build develop`. changed it anyway